lib/api.sh

```diff
@@ -3,25 +3,48 @@
 declare -A API_MSG
 API_MSG['error']="An Error occured while running request...."
 API_MSG['not_found']="Request not found"
+API_MSG['unauthorized']="No Authorization!"
```

Comment:
I would work with http code. example :
API_MSG['500']="An Error occured while running request...."
API_MSG['404']="Request not found"
......